tests/ProfilerTest.php

```diff
@@ -78,16 +78,17 @@ public function testGetEventCollection()
     {
         $profiler = self::makeProfiler();
 
-        $this->assertIsString($profiler->uuid->getValue());
+        $this->assertIsString($profiler->getId());
         $this->assertInstanceOf(EventCollection::class, $profiler->getEventCollection());
         $this->assertCount(0, $profiler->getEventCollection()->toArray());
     }
 
     public function testNothingAtProduction()
     {
-        $profiler = self::makeProfiler();
-        $profiler->isProductionMode = true;
+        $cache = new ArrayCache();
+        $profiler = new Profiler(Profiler::VERSION_1, $cache, true);
```

Comment:
The `Profiler::$isProductionMode` variable is `private` and it should create new `Profiler` class instance and assign `isProductionMode` variable to be true on `constructor`.